README.md

```diff
@@ -21,6 +21,37 @@
 
 Native Rust implementation of [Apache Iceberg](https://iceberg.apache.org/).
 
+## RoadMap
+
+### Catalog
+
+| Catalog Type | Status      |
+|--------------|-------------|
+| Rest         | Done        |
+| Hive         | In Progress |
+| Sql          | Not Started |
+| Glue         | Not Started |
+| DynamoDB     | Not Started |
+
+### FileIO
+
+| FileIO Type | Status      |
+|-------------|-------------|
+| S3          | Done        |
+| Local File  | Done        |
+| GCS         | Not Started |
```

Comment:
Do we need to make it clear that we just to add some config values?

Reply:
LGTM. I think it's ok to say that we are powered by `Apache OpenDAL`, and it would be easy to expand to other storage services.